src/python/profiler/profiler_formatter_module.py

```diff
@@ -34,125 +31,146 @@ class ProfilerJSONDecoder (json.JSONDecoder):
         """Method return list of all available formatters.
         Formatter is every class in profiler.formatters.* ns which possesses method format
         """
         result = []
         import pkgutil
 
-        for module_loader, name, ispkg in pkgutil.iter_modules (['formatters']):
+        for module_loader, name, ispkg in pkgutil.iter_modules(['formatters']):
             try:
-                module = importlib.import_module ("formatters." + name)
-                class_ = getattr (module, name)
-                if getattr (class_, 'format') is not None and callable (getattr (class_, 'format')):
-                    result.append (name)
+                module = importlib.import_module("formatters." + name)
+                class_ = getattr(module, name)
+                if getattr(class_, 'format') is not None and callable(getattr(class_, 'format')):
+                    result.append(name)
             except:
                 pass
         return result
 
-    def convert (self, json_location, output_file=None, formatter="SimpleTableFormatter", styles=[]):
+    def convert(self, json_location, output_file=None, formatter="SimpleTableFormatter", styles=[]):
         """Converts file @ json_location to output_file (if set) using given formatter name"""
         # read file to JSON
-        #try:
-        with open (json_location, 'r') as fp:
-            jsonObj = json.load (fp, encoding="utf-8", cls=ProfilerJSONDecoder)
-        #except Exception as exception:
-        #    # return string with message on error
-        #    return str (exception)
-
-        #try:
-        # split styles fields declaration
-        styles = [value.replace ('\\n', '\n').replace ('\\t', '\t').replace ('\\r', '\r') for value in styles]
-        styles = dict (item.split (":", 1) for item in styles)
-        # grab instance and hand over styles
-        instance = ProfilerFormatter.get_class_instance (formatter)
-        instance.set_styles (styles)
-        # format json object
-        output = instance.format (jsonObj)
-        #except Exception as exception:
-        #    # return string with message on error
-        #    return str (exception)
-
-        #try:
-        # if output file is specified write result there
-        if output_file is not None:
-            with open (output_file, "w") as fp:
-                fp.write (output)
-            print '{} file generated'.format (output_file)
-        # otherwise just print result to stdout
-        else:
-            print output
-        #except Exception as exception:
-        #    # return string with message on error
-        #    return str (exception)
-
-
-        # return True on success
-        #return True
-
-
-def convert (json_location, output_file, formatter):
-  """Simple iface method for c api"""
-  fmt = ProfilerFormatter ()
-  return fmt.convert (json_location, output_file, formatter)
+        Logger.instance().info('Processing file "%s"', json_location)
```

Comment:
Do not now how the Logger works, however I prefer to simply have error messages coming from exceptions on the screen. So if anything went  wrong the Flow123d propagates the error message to its own exception. Any mechanism that hides exceptions prevents fixing bugs and leads to spurious behaviour.